### PR TITLE
add w520 ( copy of t520 )

### PR DESCRIPTION
**Pull request by theotheroracle**
###### Description of changes

as far as i can tell these laptops can be configured the same, so i've just cloned the t520 folder

###### Things done

- [ x ] Tested the changes in your own NixOS Configuration
- [ x ] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -139,6 +139,7 @@
       lenovo-thinkpad-t490 = import ./lenovo/thinkpad/t490;
       lenovo-thinkpad-t495 = import ./lenovo/thinkpad/t495;
       lenovo-thinkpad-t520 = import ./lenovo/thinkpad/t520;
+      lenovo-thinkpad-w520 = import ./lenovo/thinkpad/w520;
```
Comment (Mic92) [inline flake.nix:142]: Please also add this in the README.

**Comment by Mic92** [comment]
Are you still working on this?

**Comment by theotheroracle** [comment]
sorry this slipped by, i added it to the readme

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**